pkg/descheduler/evictions/evictions.go

```diff
@@ -112,14 +113,16 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, node *v1.Node,
 	}
 	if pe.maxPodsToEvictPerNode > 0 && pe.nodepodCount[node]+1 > pe.maxPodsToEvictPerNode {
 		metrics.PodsEvicted.With(map[string]string{"result": "maximum number reached", "strategy": strategy, "namespace": pod.Namespace}).Inc()
-		return false, fmt.Errorf("Maximum number %v of evicted pods per %q node reached", pe.maxPodsToEvictPerNode, node.Name)
+		metrics.PodsEvictedSuccess.With(map[string]string{"strategy": strategy}).Inc()
```

Comment:
Why is `PodsEvictedSuccess` being incremented when maxPodsToEvictPerNode has been reached? Should this be `TotalPodsSkipped`?

Reply:
Thanks for identifying this, I'll move `PodsEvictedSuccess` to a different and more appropriate place. I might have misinterpreted it's usage here.